pkg/snapshot/restorer/restorer.go

```diff
@@ -57,7 +57,11 @@ import (
 )
 
 const (
-	etcdDialTimeout = time.Second * 30
+	etcdConnectionTimeout                     = 30 * time.Second
+	etcdCompactTimeout                        = 2 * time.Minute
+	etcdDefragTimeout                         = 5 * time.Minute
+	periodicallyMakeEtcdLean                  = 10
```

Comment:
> The current name does not specify delta snapshots anywhere. Why not call it etcdCompactDefragIntervalNumDeltaSnapshots instead, just as a suggestion?

IMO, `etcdCompactDefragIntervalNumDeltaSnapshots` it is very large variable name and as it is embedded etcd, if you embedded as well then variable name will become more large i.e `embeddedEtcdCompactDefragIntervalNumDeltaSnapshots` 

>  It indicates that a compact+defrag is performed, rather than a generic make lean term, 
yeah, I deliberately choose generic term, `makeEtcdLean` as symbol or name for abstraction, How it should be done or functionality that shouldn't be part of variable name.

> and also talks about the interval in terms of num delta snapshots intervals. WDYT?

yeah, this I can try to include.